tests/__fixtures__/cardanoSignTransaction.js

```diff
@@ -328,19 +348,41 @@ export default {
                 signingMode: CardanoTxSigningMode.ORDINARY_TRANSACTION,
             },
             result: {
-                hash: '73e09bdebf98a9e0f17f86a2d11e0f14f4f8dae77cdf26ff1678e821f20c8db6',
+                hash: '4fcd4532bb0a9dfff2368e60be80d46819a92a9acfb2c64a7bf5975040789bac',
                 witnesses: [
                     {
                         type: 0,
                         pubKey: '89053545a6c254b0d9b1464e48d2b5fcf91d4e25c128afb1fcfc61d0843338ea',
                         signature:
-                            'da07ac5246e3f20ebd1276476a4ae34a019dd4b264ffc22eea3c28cb0f1a6bb1c7764adeecf56bcb0bc6196fd1dbe080f3a7ef5b49f56980fe5b2881a4fdfa00',
+                            '7223909207fc1fa0c6125ffd4fbf7e1d845b7ca77078de7705aa34e4b6f416bb2efaaadfa022a91437ba4f33875b9a688f78d6fb30542b59417650a169afde01',
                         chainCode:
                             '26308151516f3b0e02bb1638142747863c520273ce9bd3e5cd91e1d46fe2a635',
                     },
                 ],
                 auxiliaryDataSupplement: undefined,
+                isNetworkIdIncludedInTxBody: true,
             },
+            legacyResults: [
+                {
+                    // networkId not serialized in tx body
```

Comment:
Do we want legacyResults like this for all test cases? Because that would make the file much longer and less readable, but without them, all the other cases fail when executed with an older firmware (eg. `2.4.2`).

Reply:
Ok, added.